### PR TITLE
Update to Workflow file and Unit Tests

### PR DESCRIPTION
**Pull request by mcpu-cvx**
This PR sees updates to AzManagers Unit Tests and Workflow file to link it to the new Azure Sub.

### CODE REVIEW

File: .github/workflows/Tests.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        julia-version: [1.0, 1.5]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1.0.0
+      - uses: azure/login@v1.1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: Run Az CLI script
+        run: |
+          az group create -l southcentralus -n "${{ matrix.os }}-${{ matrix.julia-version }}-${{ github.run_id }}"
```
Comment (samtkaplan) [inline .github/workflows/Tests.yml:17]: nice!

**Review by samtkaplan — COMMENTED** [review]
looks good to me.  Thanks!!  I didn't click "approve" yet since the tests are still failing.

File: .github/workflows/Tests.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - uses: julia-actions/julia-buildpkg@latest #change lastest to master
+      - uses: julia-actions/julia-runtest@latest #change lastest to master
```
Comment (scottmishra) [inline .github/workflows/Tests.yml:22]: before this step will want an ssh-keygen step, this action exists that ensure that the image has ssh-keygen installed: https://github.com/shimataro/ssh-key-action, which should make the step pretty easy

File: test/runtests.jl [hunk 3]
```diff
@@ -1,5 +1,128 @@
-using Distributed, AzManagers, Test
+using Distributed, AzManagers, Random, Test, HTTP, AzSessions, JSON
 
-@testset "AzManagers" begin
-    @test true
+template = "cbox02"
```
Comment (scottmishra) [inline test/runtests.jl:3]: We will need to run the template create steps here before we run the tests, i believe that @samtkaplan has put the necessary steps in the docs

`myscaleset = AzManagers.build_sstemplate("myvm",
    subscriptionid       = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
    location             = "southcentralus",
    resourcegroup        = "my-resource-group",
    vnet                 = "my-vnet",
    subnet               = "my-vnets-subnet",
    imagegallery         = "my-image-gallery",
    imagename            = "my-image-name",
    skuname              = "Standard_D2s_v3")
AzManagers.save_template_scaleset("myscaleset", myscaleset)`

**Review by jkwashbourne — APPROVED** [review]

**Pull request closed without merging**